### DISCUSSION TITLE
Auto-hide left sidebar

### DISCUSSION THREAD

**Discussion by poozerlang** [Godiesc/Chameleons-Beauty#14] [Q&A]
This theme is great, good work.  I'm just wondering how I would go about auto-hiding the sidebar and have it expand on hover to also show the text next to the folder icons.

There is just so much code here, and I can't even tell where to make these adjustments.

Thanks.

**Comment by Godiesc** [comment]
hi,
for that you should redone the code cause is a complete new behavior.
